R/parallel.R

```diff
@@ -0,0 +1,37 @@
+#' Get an approximate number of free cores.
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' @return the approximate number of free cores, which is an integer between 1 and one less than
+#' the total cores.
+#'
+#' @details This uses the maximum load average at 1, 5 and 15 minutes on Linux and Mac
+#' machines to approximate the number of busy cores. For Windows, the load percentage is
+#' multiplied with the total number of cores.
+#' We then subtract this from the number of all detected cores. One additional core
+#' is not used for extra safety.
+h_free_cores <- function() {
+  all_cores <- parallel::detectCores(all.tests = TRUE)
```

Comment:
because of this can you please add the `parallel` package to the Imports in DESCRIPTION and import a single function (not this one, just take the first alphabetically that pops up in RStudio when you type `parallel::`) in `mmrm-package.R` ? thx